src/RoatpProviderModeration/SFA.DAS.RoatpProviderModeration.Application/InnerApi/Requests/PatchProviderRequest.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Web;
+using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.RoatpProviderModeration.Application.InnerApi.Requests
+{
```

Comment:
please put in coverage for the PatchUrl and the building from the ProviderUpdateModel